ts/store/middlewares/analytics.ts

```diff
@@ -415,14 +415,19 @@ const trackAction = (mp: NonNullable<typeof mixpanel>) => (
     case getType(updateNotificationsInstallationToken):
     // bonus vacanze
     case getType(loadAllBonusActivations.request):
-    case getType(loadAllBonusActivations.success):
-    case getType(loadAvailableBonuses.success):
     case getType(loadAvailableBonuses.request):
+    case getType(loadAvailableBonuses.success):
     case getType(checkBonusVacanzeEligibility.request):
     case getType(cancelBonusVacanzeRequest):
     case getType(storeEligibilityRequestId):
       return mp.track(action.type);
 
+    // eslint-disable-next-line
```

Comment:
This line can be remove if we do what it suggests
_ESLint: This case's code block is the same as the block for the case on line 204.(sonarjs/no-duplicated-branches)_

👇🏼 
```typescript
    // bonus vacanze activation load success
    // wallets success / services load requests
    case getType(loadAllBonusActivations.success):
    case getType(loadServicesDetail):
      return mp.track(action.type, {
        count: action.payload.length
      });
```

Reply:
Fix PR has suggested ✅